src/pages/dmt5/_dbanner.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const Wrapper = styled.div`
+    padding: 0 0 0 12rem;
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    height: 40rem;
+    width: 100%;
+    border-top: 1px solid rgba(151, 151, 151, 0.2);
+    background-color: var(--color-black);
+
+    @media (max-width: 800px) {
+        flex-direction: column;
+        height: 414px;
+    }
+`
+const BannerWrapper = styled.div`
+    position: relative;
+    width: 100%;
+`
+const ImageWrapper = styled(Flex)`
+    position: absolute;
+    width: 50rem;
+    top: -2rem;
+    right: 12rem;
+    z-index: 1;
+    height: 100%;
+
+    div {
+        width: 100%;
+    }
+
+    @media (max-width: 1350px) {
+        width: 30rem;
+    }
+    @media (max-width: 800px) {
+        height: auto;
+        width: 286px;
+        text-align: center;
+        margin: 0 auto;
+    }
+`
+
+const TextWrapper = styled.div`
+    margin-top: 8rem;
+
+    @media (max-width: 800px) {
+        margin-top: 35px;
+        margin-bottom: 40px;
+    }
+`
+const DemoButton = styled.div`
```

Comment:
can we change `DemoButton` to something like `DemoButtonWrapper` or `ButtonContainer` since it's a div?